### DISCUSSION TITLE
Save to obsidian

### DISCUSSION THREAD

**Discussion by jonwash** [danielmiessler/Fabric#699] [Q&A]
New to WSL, and command line in general. I'm not sure how to set the fabric output path to my obsidian vault. Want to try to navigate to ~/.config/fabric/.env it returns permission denied. Have I missed something in the documentation?

**Comment by AO12612** [comment]
hi.
the tilde in a linux CLI refers to the user home directory. a permission denied means you most probably are not logged in as the user that owns that home.
I am not familiar with WSL, but I know that your windows user and the WSL user are separate .
for a simple overview (but from a non fabric related question):
https://superuser.com/a/1791389

**Comment by patonw** [comment]
The `~/.config/fabric/.env` actually refers to a file named `.env` in the `.config/fabric` sub-directory of your home folder in WSL. Navigate to the folder in a command shell rather than windows explorer and create the file there.
